web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts

```diff
@@ -275,14 +275,16 @@ export class ServerMapDiagramWithCytoscapejs extends ServerMapDiagram {
                 map(([nodes, _]: {[key: string]: any}[][]) => {
                     const edges = addedEdgeList.map((edge: {[key: string]: any}) => {
                         const {from, to, key, totalCount, hasAlert, isMerged} = edge;
+                        const originalLinkData = serverMapData.getLinkData(key);
+                        const responseInfo = this.getResponseInfo(originalLinkData);
 
                         return {
                             data: {
                                 id: key,
                                 source: from,
                                 target: to,
                                 isMerged,
-                                label: totalCount.toLocaleString(),
+                                label: totalCount.toLocaleString() + responseInfo,
```

Comment:
Use template literals when concatenating strings.
```suggestion
                                label: `${totalCount.toLocaleString()}${responseInfo}`,
```